fusor-ember-cli/app/routes/subscriptions/credentials.js

```diff
@@ -2,6 +2,24 @@ import Ember from 'ember';
 
 export default Ember.Route.extend({
 
+  setupController: function(controller, model) {
+    controller.set('model', model);
+
+    var sessionPortal = this.modelFor('subscriptions');
+    var upstream_consumer_uuid = this.modelFor('deployment').get('upstream_consumer_uuid');
+    // check if org has upstream UUID using Katello V2 API
+    var orgID = this.modelFor('deployment').get('organization.id')
+    var url = '/katello/api/v2/organizations/' + orgID;
```

Comment:
@bbuckingham, this is using katello API v2 to get owner_details.upstreamConsumer.uuid